crypto/jwx_mitzxnuts.go

```diff
@@ -0,0 +1,10 @@
+package crypto
+
+func GetSupportedAlgorithms() []string {
+	var algorithms []string
+	for _, curr := range supportedAlgorithms {
```

Comment:
Don't know if this is needed but could have been done with `copy()`?

Reply:
This code is no longer needed, as it was part of the DCR metadata request.